apps/heft/src/stages/CleanStage.ts

```diff
@@ -36,16 +37,43 @@ export interface ICleanStageOptions {
  */
 export interface ICleanStageContext extends IStageContext<CleanStageHooks, ICleanStageProperties> {}
 
+interface ICleanConfigurationJson {
+  pathsToDelete: string[];
+}
+
 export class CleanStage extends StageBase<CleanStageHooks, ICleanStageProperties, ICleanStageOptions> {
   public constructor(heftConfiguration: HeftConfiguration, loggingManager: LoggingManager) {
     super(heftConfiguration, loggingManager, CleanStageHooks);
   }
 
-  protected getDefaultStageProperties(options: ICleanStageOptions): ICleanStageProperties {
+  protected async getDefaultStagePropertiesAsync(
+    options: ICleanStageOptions
+  ): Promise<ICleanStageProperties> {
+    const cleanConfigurationFileLoader: ConfigurationFile<ICleanConfigurationJson> = new ConfigurationFile<
```

Comment:
Can we separate the "declaration of a config file" from the "loading of the config file"?  For example, suppose the code here reduced to something like:

```ts
const cleanConfigurationFileLoader: ConfigurationFile<ICleanConfigurationJson> 
  = HeftConfigFiles.clean.loadAsync(myPath);
```

Then if I'm trying to find the code that loads `clean.schema.json`, I can go to `HeftConfigFiles.ts`  and all the declarations are nicely arranged there.